config/locales/da-DK.yml

```diff
@@ -35,15 +35,15 @@ da-DK:
   devise:
     mailer:
       confirmation_instructions:
-        confirm_link_msg: "Du kan bekræfte din konto email for linket herunder:"
+        confirm_link_msg: "Du kan bekræfte din kontos email gennem linket herunder:"
         confirm_account_link: "Bekræft min konto"
       reset_password_instructions:
         request_reset_link_msg: "Der er nogle der har anmodet om et link til at ændre dit kodeord. Det kan du gøre gennem linket nedenfor."
         password_change_link: "Ændre mit kodeord."
         ignore_mail_msg: "Hvis du ikke anmodede om dette, ignorer venligst denne email."
-        no_changes_msg: "Din kodeord vil ikke ændres indtil du går ind på linket ovenfor og laver et nyt et."
+        no_changes_msg: "Dit kodeord vil ikke ændres indtil du går ind på linket ovenfor og opretter et nyt."
```

Comment:
`Dit kodeord vil ikke blive ændret indtil du...`

I'm not sure about the context here, it sounds a bit weird to "make a new" password, usually you would say that you change a password.